xss_scanner/xss_scanner.py

```diff
@@ -0,0 +1,113 @@
+import subprocess
+from splinter import Browser
+from fuzzywuzzy import fuzz
+
+from artemis import load_risk_class, utils
+from artemis.binds import TaskStatus, TaskType
+from artemis.module_base import ArtemisBase
+from artemis.task_utils import get_target_host
+from karton.core import Task
+
+from extra_modules_config import ExtraModulesConfig
+
+logger = utils.build_logger(__name__)
+
+# Ignore for now
+PAYLOADS = {"test_payload"}
+FUZZY_LEVEL = 95
+CRAWLING_LEVEL = 1
+
+
+class XssScanner(ArtemisBase):  # type: ignore
+    identity = "xss-scanner"
+
+    def prepare_crawling_result(self, output_str: str) -> set:
```

Comment:
could you add a docstring and an unittest for this function? It has a lot of logic that's not obvious if someone is not familiar with xss-scanner output syntax